src/Http/Controllers/StatsController.php

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helper. It's overkill to me.